### ISSUE TITLE
Unable to select models in VSCode Insiders

### ISSUE THREAD

**Issue by accesswatch** [microsoft/vscode#246930]
Type: <b>Bug</b>

1. I removed all settings and extensions from my copy of VSCode Insiders.
2. I launched the editor and turned on screen reader mode.
3. I pressed ctrl + alt + i and tabbed to the mode button. This button works.
4. I right arrowed to the model selecttor and attempted to invoke it.
5. Nothing happens with the keyboard.

VS Code version: Code - Insiders 1.100.0-insider (1b000ef82b3c3082833f6b04e050bc0e4e12ae3b, 2025-04-18T05:31:28.011Z)
OS version: Windows_NT x64 10.0.26200
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 5 Microsoft Surface (R) Edition (12 x 2196)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|undefined|
|Memory (System)|7.45GB (1.44GB free)|
|Process Argv|--crash-reporter-id 0262c457-42ea-4acd-a36e-66ade47d9f56|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (2)</summary>

Extension|Author (truncated)|Version
---|---|---
copilot|Git|1.304.1517
copilot-chat|Git|0.27.2025041802

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyt551:31249597
vscod805cf:30301675
c4g48928:30535728
2i9eh265:30646982
962ge761:30841072
dsvsc014:30777825
dsvsc015:30821418
2e7ec940:31000449
pythontbext0:30879054
cppperfnew:30980852
dwnewjupyter:31046869
pythonrstrctxt:31093868
nativeloc1:31118317
e80f6927:31120813
dwcopilot:31158714
c3hdf307:31184662
6074i472:31201624
dwoutputs:31242946
customenabled:31248079
hdaa2157:31222309
copilot_t_ci:31222730
g012b348:31231168
pythoneinst12:31251391
treesitterini:31278752
bgtreat:31252123
c7cif404:31283753
31787653:31262186
3e8i5726:31271747
pythonpulldiag:31287486
6gi0g917:31259950
61ibe868:31282443
996jf627:31264550
pythonrdcb7:31268811
usemplatestapi:31288272
gh9da704:31280609
747dc170:31275146
g20af354:31278749
9hi5e907:31287812
pylancecolor:31288664
generatesymbolt:31280541
convertfstringf:31280702
gendocf:31280703

```

</details>

**Comment by accesswatch** [comment]
The hotkey to move to the next model works and an ARIA alert is fired. This is simply a problem with the context menu that used to work when invoking the button.

**Comment by meganrogge** [comment]
/duplicate [246985](https://github.com/microsoft/vscode/issues/246985)
